Backends/RasterImage/output-to-image.lisp

```diff
@@ -44,20 +44,19 @@
 
 (defmacro with-output-to-image ((stream-var image &rest options)
                                 &body body)
+  (declare (ignore image))
   (alexandria:with-gensyms (cont exit-fn enter-fn)
-    (alexandria:once-only (image)
-      `(flet ((,cont (,stream-var)
-                ,@body)
-              (,exit-fn (sheet stream)
-                (declare (ignore stream))
-                (image-mirror-image (sheet-mirror sheet)))
-              (,enter-fn (sheet stream)
-                (declare (ignore stream))
-                (when ,image
-                  (setf (image-mirror-image sheet) ,image))))
-         (declare (dynamic-extent #',cont))
-         (invoke-with-output-to-raster-image
-          #',cont #',enter-fn #',exit-fn :rgb-image :rgb-image ,@options)))))
+    `(flet ((,cont (,stream-var)
+              ,@body)
+            (,exit-fn (sheet stream)
+              (declare (ignore stream))
+              (image-mirror-image sheet))
+            (,enter-fn (sheet stream)
+              (declare (ignore stream))
+              nil))
+       (declare (dynamic-extent #',cont))
+       (invoke-with-output-to-raster-image
+        #',cont #',enter-fn #',exit-fn :rgb-image :rgb-image ,@options))))
```

Comment:
I may have asked this before, but doesn't this break the macro?

Reply:
this macro is not exported and it is never used with image being not nil.

Reply:
reintroduced(not used) mechanism - it may be potentially useful, I'll see later during work on the renderer